config/platforms/platform_solaris.h.in

```diff
@@ -11,7 +11,7 @@
 #define DUK_USE_BYTEORDER 3
 #endif
 #else  /* DUK_F_OLD_SOLARIS */
-#include <ast/endian.h>
+#include <sys/param.h>
```

Comment:
`<sys/param.h>` is already included below regardless of `DUK_F_OLD_SOLARIS` so I don't think any include is needed if sys/param.h indeed provides the necessary byte order defines.